xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 12.5.0
+JSONRPC_VERSION 12.5.1
```

Comment:
As a non-breaking addition to the API this is a minor version bump e.g. 12.7.0

Reply:
Ha yes 12.6.0, I obviously can't count this morning!
To be fair I had also been looking at another API change PR so may have bumped it in my head :)

Reply:
Merged that PR so 12.7.0 is right now after rebase.